src/components/QuestionSubmissionForm.tsx

```diff
@@ -59,10 +59,18 @@ export function QuestionSubmissionForm({ session }: PageProps) {
 								<div className='mt-1 flex flex-col gap-y-1'>
 									<input
 										type='text'
+										list='companies'
+										{...register('company', { required: true })}
 										id='company'
 										className='block w-full min-w-0 flex-1 rounded-md border-gray-300 focus:border-indigo-500 focus:ring-indigo-500 sm:text-sm'
-										{...register('company', { required: true })}
 									/>
+									<datalist id='companies'>
+										<option value='Facebook' />
+										<option value='Amazon' />
+										<option value='Apple' />
+										<option value='Netflix' />
+										<option value='Google' />
```

Comment:
Add Microsoft.

For Facebook - let's use the official name Meta, or maybe use something like Meta/Facebook?

Reply:
@ykdojo I have pushed new changes.